src/opm/input/eclipse/Schedule/KeywordHandlers.cpp

```diff
@@ -2429,8 +2437,12 @@ Well{0} entered with 'FIELD' parent group:
                     sicd.updateScalingFactor(outlet_segment_length, connections.segment_perf_length(segment_nr));
                 }
 
-                if (well.updateWSEGSICD(sicd_pairs) )
+                if (well.updateWSEGSICD(sicd_pairs) ) {
```

Comment:
It is not desirable to use `well` after `std::move(well)`.  You can avoid that by 
```c++
const bool well_status = well.getStatus();
```

before the line with `std::move()`

It also applies to the two occasions below.

Reply:
Thanks, will fix.